src/utils/check-file-path.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+import { escapeRegExp as escapeRe } from 'lodash';
+import { win as isWin } from 'os-family';
+import sanitizeFilename from 'sanitize-filename';
+import { diffChars } from 'diff';
+
+
+function createForbiddenCharsDescription (filePath, diff) {
+    let currentIndex = 0;
+    let description  = '';
+
+    for (const change of diff) {
+        if (change.removed && (!isWin || currentIndex !== 1 || !path.isAbsolute(filePath)))
+            description += `\t"${change.value}" at index ${currentIndex + 1}\n`;
+
+        currentIndex += change.count;
+    }
+
+    return description;
+}
+
+export default function (filePath) {
+    filePath = filePath.replace(new RegExp(escapeRe(path.win32.sep), 'g'), path.posix.sep);
+
+    const sanitizedPath = filePath
```

Comment:
You can simplify to `const sanitizedPath = filePath.sanitizeFilename(filePath);` because you already replaced a separator.

Same for https://github.com/DevExpress/testcafe/pull/2820/files#diff-03b99b1b96a2a21029660e35954c8661R9

Reply:
No, because `sanitizeFilename` will remove `/` from the path.